src/forms/SpatialBoundsField.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
+import React, { useEffect, useRef } from "react";
```

Comment:
I plan to remove the whole `forms` directory and its files later on when we finish refactoring the new management client. So in this case, it's better to migrate the SpatialBoundsField.tsx to the form directory. I will keep it in my to-do list and do it later.

Reply:
done